src/common/output/fix.rs

```diff
@@ -54,4 +54,13 @@ impl FixOutput {
             println!();
         }
     }
+
+    /// Prints no files found message
+    pub fn print_no_files_message(&self) {
```

Comment:
```suggestion
    pub fn print_nothing_to_fix(&self) {
```

Reply:
I have renamed the function and removed that unnecessary `\n`, thanks for CR :)